vmdk_plugin/plugin.go

```diff
@@ -281,7 +321,7 @@ func (d *vmdkDriver) Mount(r volume.MountRequest) volume.Response {
 		isReadOnly = true
 	}
 
-	mountpoint, err := d.mountVolume(r.Name, isReadOnly)
+	mountpoint, err := d.mountVolume(r.Name, status["fstype"].(string), isReadOnly)
```

Comment:
In "error prone" I meant whatever bug we may have in KV management may lead to an attempt to mount a wrong FS type. Nothing specific, just a vague feeling that the FS type in KV is redundant. 

But yes, you right - I forgot we switched to syscall from invoking mkfs executable so tracking it in KV and feeding ro syscall.Mount is a good choice